src/lib/attestationData.ts

```diff
@@ -0,0 +1,36 @@
+import { HaLoNoncePCD } from "@pcd/halo-nonce-pcd";
+
+export function attestationText(pcd: HaLoNoncePCD): string | undefined {
```

Comment:
Don't need to change anything in this PR, but in the future this should be fetched from the backend